Sources/Mantis/Protocols/CropViewControllerDelegate.swift

```diff
@@ -18,19 +18,24 @@ public protocol CropViewControllerDelegate: AnyObject {
     func cropViewControllerDidBeginResize(_ cropViewController: CropViewController)
     func cropViewControllerDidEndResize(_ cropViewController: CropViewController, original: UIImage, cropInfo: CropInfo)
     
-    @available(*, deprecated, message: "Use cropViewControllerDidImageTransformed(_ cropViewController: CropViewController, transformation: Transformation) instead")
-    func cropViewControllerDidImageTransformed(_ cropViewController: CropViewController)
-    
     func cropViewControllerDidImageTransformed(_ cropViewController: CropViewController, transformation: Transformation)
     
     func cropViewController(_ cropViewController: CropViewController, didBecomeResettable resettable: Bool)
+    
+    func cropViewControllerDidUpdateEnableStateForUndo(_ enable: Bool)
+    
+    func cropViewControllerDidUpdateEnableStateForRedo(_ enable: Bool)
+    
+    func cropViewControllerDidUpdateEnableStateForReset(_ enable: Bool)
 }
```

Comment:
I checked these three functions again. I think we need to one more time renaming like below to keep code design consistent. Usually when designing delegate, we need to pass instance itself back to its delegate.

```Swift
    func cropViewController(_ cropViewController: CropViewController, didUpdateEnableStateForUndo enable: Bool)

    func cropViewController(_ cropViewController: CropViewController,  didUpdateEnableStateForRedo enable: Bool)

    func cropViewController(_ cropViewController: CropViewController,  didUpdateEnableStateForReset  enable: Bool)
```

Reply:
changes made.